packages/app-project/src/shared/components/CollectionsModal/components/SelectCollection/SelectCollection.js

```diff
@@ -10,10 +12,23 @@ function SelectCollection ({
   onSubmit,
   selected
 }) {
+  const [searchText, setSearchText] = useState('')
   const { t } = useTranslation('components')
   const dropProps = {
     trapFocus: false
   }
+  function onTextChange(text) {
+    onSearch({
+      favorite: false,
+      current_user_roles: 'owner,collaborator,contributor',
+      search: text.length > 3 ? text : undefined
+    })
+    setSearchText(text)
+  }
+  const options = searchText.length > 3 ?
+    collections :
+    collections.filter(collection => collection.display_name.includes(searchText))
```

Comment:
I think `String.includes()` is case sensitive. Maybe downcase both strings before matching?

Reply:
```suggestion
    collections.filter(collection => collection.display_name.toLowerCase().includes(searchText.toLowerCase()))
```